src/components/DeleteAlert.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const DeleteAlert = () => {
+  return (
+    <div
+      className="p-4 mb-4 text-sm text-red-800 rounded-lg bg-red-50 dark:bg-gray-800 dark:text-red-400"
+      role="alert"
+    >
+      <span className="font-medium">Removed!</span>
+      You removed a expense group
```

Comment:
here you can only use one warning message, this is how it shows when a user removes a group:
![Screenshot 2024-09-24 220544](https://github.com/user-attachments/assets/b1742fc6-1df2-442b-a9de-125b8a3d2c64)
I feel like it's doesn't give it a nice view.